studiengaenge/lehramt_info.tex

```diff
@@ -8,23 +8,30 @@ \section{Lehramt Informatik}
 	\includegraphics[width=3.1cm]{bilder/xkcd_responsible_behaviour_4.png}
 }
 
-Wenn man  Lehramt Informatik studiert, sind zwei Fälle zu unterscheiden: Euer anderes
-Fach ist Mathematik oder euer anderes Fach ist etwas Anderes. Diese
-Unterscheidung rührt daher, dass alle Lehramtskandidat\_innen
-Informatik etwas Mathematik können sollen und ihr diese, solltet ihr
-Mathematik studieren, natürlich automatisch lernt -- und solltet ihr
-etwas Anderes studieren, eher nicht.
+Wenn ihr den 50\%-Bachelor Informatik studiert, sind zwei Fälle zu unterscheiden: Ihr hört in
+eurem zweiten Fach Mathematik Veranstaltungen oder ihr tut es nicht. Diese
+Unterscheidung rührt daher, dass ihr in der Informatik auf jeden Fall etwas
+Mathematik können solltet, wenn ihr in eurem anderen fach Mathe hört habt ihr das aber
+schon da abgedeckt und ihr könnt euch in dieser Hälfte eures Bachelors ganz auf die Informatik konzentrieren.
 
-Für den Fall, dass ihr Mathematik studiert, ist es fast ganz einfach:
-Ihr studiert Mathematik, wie es sich für das Lehramt gehört und hört auf
-jeden Fall noch die Vorlesung \hyperref[info1]{„Einführung in die
-  Praktische Informatik“ (Info 1)}. Das ist wichtig, denn das ist eure
-Orientierungsprüfung in Informatik: diesen Schein zu bekommen. Auch
-solltet ihr den Programmierkurs machen. Im zweiten Semester macht ihr
-dann in Informatik die „Einführung in die Theoretische Informatik“ und
-„Algorithmen und Datenstrukturen“, im dritten und vierten Semester dann die
-„Einführung in die Technische Informatik“, „Betriebssysteme und
-Netzwerke“, die erste Fachdidaktik und eure Zwischenprüfung (s.\,u.)
+Auf jeden Fall Hört ihr im Ersten Semester \hyperref[info1]{„Einführung in die Praktische Informatik“ (Info 1)},
+die ist Wichtig da es eure Orientierungsprüfung in Informatik ist, und den Programmierkurs.
+Im zweiten Semester solltet ihr dann in Informatik die „Einführung in die Theoretische Informatik“
+\footnote{„Theo“ kann zu verwirungen mit Physikern führen da diese das als Theoretische Physik verstehen}
+und „Algorithmen und Datenstrukturen“ (AlDa) hören. Im Dritten Semester macht sich der Unterschied 
+ob ihr in eurem Zweiten Fach Mathe hört oder nicht da ihr hier „Mathematik für Informatiker 1“ (MafIn 1) sollt,
+wenn ihr aber bereits andere Mathe Veranstaltungen Bestanden habt könnt ihr beim Prüfungsausschuss beantragen
+das ihr statt dessen ein weiteres Info Modul, aus dem Bereich der Wahlpflicht Veranstaltungen, hören könnt.
```

Comment:
Über diesen Satz sollte man nochmal drüber gehen. Im ersten Halbsatz fehlt vermutlich irgendetwas wie „bemerkbar“. Ich würde das erste Komma durch einen Punkt oder ein Semikolon ersetzen. In Zeile 24 schreibt man „das“ glaub ich mit zwei s.